python/tests/transformers/keras_transformer_test.py

```diff
@@ -16,32 +16,76 @@
 import os
 
 from keras.models import Sequential
-from keras.layers import Dense, Activation
+from keras.layers import Dense, Activation, Reshape
 from keras.initializers import glorot_uniform
 
 from sparkdl.transformers.keras_transformer import KerasTransformer
-from ..tests import SparkDLTestCase
+from ..tests import SparkDLTempDirTestCase
 
-class KerasTransformerTest(SparkDLTestCase):
 
-    # TODO(sid): How to create static class variables in Python? Should I just put these in the
-    # constructor?
-    NUM_FEATURES = 10
+class KerasTransformerTest(SparkDLTempDirTestCase):
+    # Seed used for random-number generation in tests
     RANDOM_SEED = 997
 
-    def _loadNumpyData(self, num_examples, num_features):
+    def test_keras_transformer_single_dim(self):
         """
-        Construct and return a 2D numpy array of shape (num_examples, num_features) corresponding
-        to a dataset of one-dimensional examples.
+        Test that KerasTransformer correctly handles single-dimensional input data.
         """
-        np.random.seed(self.RANDOM_SEED)
-        return np.random.randn(num_examples, num_features)
+        # Construct a model for simple binary classification (with a single hidden layer)
+        model = Sequential()
+        input_shape = [10]
+        model.add(Dense(units=10, input_shape=input_shape,
+                        bias_initializer=self._getKerasModelWeightInitializer(),
+                        kernel_initializer=self._getKerasModelWeightInitializer()))
+        model.add(Activation('relu'))
+        model.add(Dense(units=1, bias_initializer=self._getKerasModelWeightInitializer(),
+                        kernel_initializer=self._getKerasModelWeightInitializer()))
+        model.add(Activation('sigmoid'))
+        # Compare KerasTransformer output to raw Keras model output
+        self._test_keras_transformer_helper(model, model_filename="keras_transformer_single_dim")
 
-    def getInputDF(self, sqlContext, inputCol, idCol):
-        """ Return a DataFrame containing a long ID column and an input column of arrays. """
-        x_train = self._loadNumpyData(num_examples=20, num_features=self.NUM_FEATURES)
-        train_rows = [{idCol : i, inputCol : x_train[i].tolist()} for i in range(len(x_train))]
-        return sqlContext.createDataFrame(train_rows)
+    def test_keras_transformer_multi_dim(self):
```

Comment:
name it `test_keras_transformer_2dim` so it's clear it doesn't work for 3+ dimensions